doc/cephfs/fuse.rst

```diff
@@ -1,53 +1,92 @@
-=======================
-Mount CephFS using FUSE
-=======================
+========================
+ Mount CephFS using FUSE
+========================
 
-Before mounting Ceph via "Filesystem in USErspace" (FUSE), ensure that the client
-host has a copy of the Ceph configuration file and a keyring with CAPS for the
-Ceph metadata server.
+Pre-requisite
+-------------
 
-#. From your client host, copy the Ceph configuration file from the monitor host 
-   to the ``/etc/ceph`` directory. :: 
+Before mounting Ceph via "Filesystem in USErspace" (FUSE), ensure that the
+client host (where CephFS has to mounted and used) has a copy of the Ceph
+configuration file (i.e. ``ceph.conf``) and a keyring of CephX user that has
+CAPS for the Ceph MDS. Both of these files must be present on the host where
+the Ceph MON resides.
 
+#. Copy the Ceph conf from the MON host to client host at ``/etc/ceph``::
+
+	# on client host
 	sudo mkdir -p /etc/ceph
-	sudo scp {user}@{server-machine}:/etc/ceph/ceph.conf /etc/ceph/ceph.conf
+	ssh {user}@{mon-host} "sudo ceph config generate-minimal-config" | sudo tee /etc/ceph/ceph.conf
+
+#. Ensure that the conf has appropriate permissions::
```

Comment:
You're talking about the configuration file but the chmod command refers to the mount point.